[2024-07-07T13:09:43.892+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T13:09:43.928+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final.starting manual__2024-07-07T13:09:38.251967+00:00 [queued]>
[2024-07-07T13:09:43.942+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final.starting manual__2024-07-07T13:09:38.251967+00:00 [queued]>
[2024-07-07T13:09:43.949+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-07T13:09:44.056+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): starting> on 2024-07-07 13:09:38.251967+00:00
[2024-07-07T13:09:44.070+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=55) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T13:09:44.073+0000] {standard_task_runner.py:63} INFO - Started process 57 to run task
[2024-07-07T13:09:44.074+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final', 'starting', 'manual__2024-07-07T13:09:38.251967+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpz1jwjqsa']
[2024-07-07T13:09:44.077+0000] {standard_task_runner.py:91} INFO - Job 153: Subtask starting
[2024-07-07T13:09:44.223+0000] {task_command.py:426} INFO - Running <TaskInstance: Final.starting manual__2024-07-07T13:09:38.251967+00:00 [running]> on host 98a488506102
[2024-07-07T13:09:44.695+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final' AIRFLOW_CTX_TASK_ID='starting' AIRFLOW_CTX_EXECUTION_DATE='2024-07-07T13:09:38.251967+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-07T13:09:38.251967+00:00'
[2024-07-07T13:09:44.699+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T13:09:44.701+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-07T13:09:44.703+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Chuẩn bị cào dữ liệu"']
[2024-07-07T13:09:44.723+0000] {subprocess.py:86} INFO - Output:
[2024-07-07T13:09:44.730+0000] {subprocess.py:93} INFO - Chuẩn bị cào dữ liệu
[2024-07-07T13:09:44.731+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-07T13:09:44.733+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T13:09:44.882+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Final, task_id=starting, run_id=manual__2024-07-07T13:09:38.251967+00:00, execution_date=20240707T130938, start_date=20240707T130943, end_date=20240707T130944
[2024-07-07T13:09:45.015+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T13:09:45.074+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-07T13:09:45.157+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
